### PR TITLE
Add support to Minetest Bower

### PR DESCRIPTION
**Pull request by diogo-gomes**
Add support to [Minetest Bower](https://minetest-bower.herokuapp.com).

- Update(fix) bower.json
- Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,27 @@ Currently includes:
 
 **Version:** Unreleased ([change log](CHANGELOG.md))
 
+
+---
+## Install
+
+### Minetest Bower
+##### (recommended)
```
Comment (LNJ2) [inline README.md]: I wouldn't use a newline here

File: bower.json [hunk 2]
```diff
@@ -1,25 +1,27 @@
 {
-    "name": "Creatures MOB-Engine",
-    "description": "A Mod(pack) for Minetest that provides a MOB-Engine and adds several creatures to the game.\n",
+    "name": "mob-engine",
+    "description": "Mob Engine is a mod(pack) for Minetest game that provides a mob engine and adds several mobs to the game.",
+    "homepage": "https://github.com/minetest-mods/mob-engine",
+    "authors": [
+        "BlockMen - author <blockmen2015@gmail.com>"
```
Comment (LNJ2) [inline bower.json]: I think this should be changed now ...

**Comment by sofar** [comment]
I'd make the commit messages in English.

File: bower.json (hunk 2 above)
Comment (diogo-gomes) [inline bower.json]: But BlockMen is original author. What do you recommend?

**Comment by diogo-gomes** [comment]
@sofar, you are right, my mistake.

I'll fix it with squash and merge.

I'll pay more attention next time.

File: bower.json (hunk 2 above)
Comment (LNJ2) [inline bower.json]: I'd add him and the new maintainers.

**Pull request merged**